scrapers/TheFlourish.yml

```diff
@@ -0,0 +1,41 @@
+name: The Flourish
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - theflourishxxx.com/trailers/
+      - theflourishamateurs.com/trailers/
+      - theflourishfetish.com/trailers/
+      - theflourishpov.com/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[contains(@class, 'modelDetails')]
+    scene:
+      Title: //div[contains(@class,'title')]/h2/text()
+      URL: //link[@rel="canonical"]/@href
+      Details: $details/div[@class='description']/p/text()
```

Comment:
Either remove the text() part to leave the whole p as a selector `$details/div[@class='description']/p` or do a post process and concat the texts to get the full description. For scenes with a long description you are only getting part of it
sample url `https://tour.theflourishxxx.com/trailers/The-Pros-Episode-01-Armani-Black.html`